src/raftstore/store/peer.rs

```diff
@@ -1591,7 +1591,8 @@ impl Peer {
         let last_ready_read_count = self.raft_group.raft.ready_read_count();
 
         let id = self.pending_reads.next_id();
-        let ctx: [u8; 8] = unsafe { mem::transmute(id) };
+        let ctx = id.to_bytes();
+        // TODO: Replace with to_ne_bytes() here if we upgrade rustc to 1.30 or above
```

Comment:
I think we can move to Rust 2018 ASAP.

Reply:
Agreed; though this method is also available in 2015 :)

Reply:
I'll make a note about this for the next Rust upgrade.